src/Discord.Net.Commands/ModuleBase.cs

```diff
@@ -4,8 +4,15 @@
 
 namespace Discord.Commands
 {
+    /// <summary>
+    ///     The base class that command modules inherit from where the context is of type <see cref="ICommandContext"/>.
+    /// </summary>
     public abstract class ModuleBase : ModuleBase<ICommandContext> { }
```

Comment:
Not a fan of this one.

Reply:
Perhaps `Provides a base class for a command module to inherit from` or something along the lines?